aiida/storage/psql_dos/backend.py

```diff
@@ -331,14 +333,17 @@ def delete_nodes_and_connections(self, pks_to_delete: Sequence[int]) -> None:
 
         session = self.get_session()
         # Delete the membership of these nodes to groups.
-        session.query(DbGroupNode).filter(DbGroupNode.dbnode_id.in_(list(pks_to_delete))
+        session.query(DbGroupNode).filter(DbGroupNode.dbnode_id.in_(list(pks_to_delete))  # type: ignore[attr-defined]
```

Comment:
It will say something like
```
Self? has no attribute "values"  [attr-defined]
```
The exact reason is not clear. But see my other reply why I opted to just go with an ignore rule for now.